packages/web-components/src/components/va-back-to-top/va-back-to-top.tsx

```diff
@@ -86,10 +87,13 @@ export class VaBackToTop {
           <div
             class={classnames({ docked: this.isDocked, reveal: this.revealed })}
           >
-            <button type="button" onClick={this.navigateToTop.bind(this)}>
-              <i aria-hidden="true" role="img"></i>
-              <span>Back to top</span>
-            </button>
+            <a href="#ds-back-to-top" onClick={this.navigateToTop.bind(this)}>
```

Comment:
In order pass the accessibility Axe check, the link would need to be in one of the following patterns which is why I added the `href="#ds-back-to-top"`: 

https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/anchor-is-valid.md#succeed

I think what I would have liked to do here is to use the href as an actual link to an element with that ID. I experimented with doing something like programmatically adding a div directly after the `body` tag like this:

```js
const backToTopEl = document.createElement('div');
backToTopEl.id = 'ds-back-to-top';
document.body.parentNode.insertBefore(backToTopEl, document.body);
```

But the click is doing more than just navigating. It's adding a focus to the first `h1` on the page and this also didn't work very well in an iframe (like Storybook). 

So I'm adding `href="#ds-back-to-top"` to make the link valid but it doesn't actually function in the way it appears. The alternative was to add `// eslint-disable-next-line anchor-is-valid` above the line.